### DISCUSSION TITLE
Positioning the Menu component with Poper.js

### DISCUSSION THREAD

**Discussion by AlexMachin1997** [tailwindlabs/headlessui#2179] [Help]
Hi everyone,

Just wondering if you can use the usePopper/Popper with headlessuis "Menu" component ? If so are there any examples ?

I know you can use it with the popover, Listbox and Combobox but wondering if you can also use for the Menu as I want the keyboard control which popover doesn't offer as far as I could see.

So far I've managed to get Popper added but for some reason it's always centered I can't seem to get it to start on the right or left correctly.

Any comments are greatly appreciated 😀

**Comment by whitespacecode** [comment]
I use Float
https://headlessui-float.vercel.app/vue/quick-start.html

**Reply by AlexMachin1997** [reply]
I should have deleted this question or at least updated it.

I got it working with popper, it worked well for my personal project

**Reply by greg-hoarau** [reply]
Note that next release of Headless UI will integrate Floating UI (see: [Tweet from Adam Wathan](https://twitter.com/adamwathan/status/1729274129264624056))

**Reply by whitespacecode** [reply]
Will be a very nice update! Thanks for the heads up
